src/Configuration.ts

```diff
@@ -94,13 +102,7 @@ function loadConfigurationFromFile(configPath: string): Configuration | {} {
 }
 
 function loadConfigurationFromEnv(): Partial<Configuration> {
-  // TODO: programmatically generate keys from values by applying camelCase to SCREAMING_SNAKE_CASE
-  const map: { [index: string]: string } = {
-    RABBITMQ_URL: 'rabbitmqUrl',
-    MONGODB_URL: 'mongodbUrl',
-    IPFS_URL: 'ipfsUrl',
-    INSIGHT_URL: 'insightUrl',
-  }
+  const map: { [index: string]: string } = createEnvToConfigurationKeyMap(defaultConfiguration)
```

Comment:
Best to properly type `createEnvToConfigurationKeyMap` and let TS infer the type of `map`.